uberduck_ml_dev/trainer/base.py

```diff
@@ -557,8 +596,11 @@ def train(self):
                 else:
                     reduced_mel_loss = mel_loss.item()
                     reduced_gate_loss = gate_loss.item()
-                reduced_loss = reduced_mel_loss + reduced_gate_loss
+                    reduced_gate_loss_batch = gate_loss_batch  # .item()
+                    reduced_mel_loss_batch = mel_loss_batch  # .item()
```

Comment:
Should we call `.detach()` on these? I think otherwise we'll compute extra gradients.

Reply:
I don't think we will have this issue?  We call loss.backward() where loss = gate_loss + mel_loss, so I'm not sure adding outputs that we don't differentiate through will add gradients.  Why not detach tho